.github/ISSUE_TEMPLATE/story.md

```diff
@@ -44,3 +44,49 @@ What else should contributors [keep in mind](https://fleetdm.com/handbook/compan
 1. 
 2. 
 -->
+
+
+## Test plan
+
+- [ ] Requires load testing TODO <!-- User story has performance implications that require load testing. Otherwise, remove this checkbox. -->
+
+## Risk assessment
+
+Risk level: Low / Medium / High TODO <!-- Choose one. -->
+
+Risk description: TODO <!-- If risk level is medium or high, explain why. If low, remove. -->
+
+#### Automated:
+
+- Fleet: Covered / Will not cover / Scoped <!-- Choose one. -->
+- QAWolf: Covered / Will not cover / Scoped <!-- Choose one. -->
+
+## Manual testing steps
+<!-- Add detailed manual testing steps for all affected user roles. -->
+
+Admin: TODO
+1. Step 1
+2. Step 2
+3. Step 3
+
+Maintainer: TODO
+1. Step 1
+2. Step 2
+3. Step 3
+
+Observer: TODO
+1. Step 1
+2. Step 2
+3. Step 3
+
+## Testing notes
+<!-- Any additional testing notes relevant to this story. -->
+
+## Tools required
+<!-- Any additional tools required (local TUF service, API testing, specific software or VM) -->
```

Comment:
this makes sense, but another way to handle it could be for Quality to document the steps they used to test, which covers this automatically.  Rather than defining a test plan ahead of time, it can be helpful (and require less total time) to document the test plan as the testing actually happens.  

For example, you could merge "Tools required" with "Manual testing steps", and have "Manual testing steps" cover the steps actually taken to QA, with the outcome.  Up to Luke